varnish/tests/pass/metrics.rs

```diff
@@ -19,4 +19,12 @@ pub struct VariousStats {
     /// A more detailed description than the above oneliner could go here.
     #[gauge(level = "debug", format = "bytes")]
     memory: AtomicU64,
+
+    /// Some arbitrary bitmap
+    #[bitmap]
+    flags: AtomicU64,
+
+    /// A bitmap with integer format override
+    #[bitmap(format = "integer")]
```

Comment:
I think the format should even be valid here?

Reply:
What do you mean? It seems like it would be technically valid to have a bitmap with format integer, even if unlikely to be useful?

Reply:
it's technically valid in the sense that the JSON presents the two things independently, but a bitmap is a bitmap, the format is meaningless. Like, a metric or a gauge can represent an amount of bytes or a duration, but of the formats we have, a bitmap can only be a bitmap.

Plus, but I think if you start `varnishstat` in `curses` mode, you will see that if the format isn't `b`, you won't get the bitfield representation

Reply:
I think that would be perfect

Reply:
Make it sane, we can release 6.0.0, not big deal